.gitignore

```diff
@@ -1 +1,3 @@
-node_modules
\ No newline at end of file
+node_modules
+package-lock.json
+package.json
```

Comment:
- [ ] Please do not add package-lock.json or package.json to `.gitignore`. These two are very important for those who might need to do any package installation to run the project locally (to run linters for example). Instead, add the files the last reviewer commented on.